[Doc] Update `Crypto` example

DarkiStar opened the pull request:
Removed redundant new() initializations

DarkiStar committed:
Update Crypto.xml

Removed redundant new() initializations

timothyqiu commented:
I think the `extends Node` and `_ready()` setup is also unrelated information. The example could be just ... code.

DarkiStar committed:
Update Crypto.xml Formating

Made easier to read and moved crypto.new() into a function.

DarkiStar committed:
Update Crypto.xml 2

DarkiStar committed:
Update Crypto.xml 3

DarkiStar committed:
Update Crypto.xml (#1)

Removed redundant new() initializations
Made easier to read and moved crypto.new() into a function.

AThousandShips reviewed (dismissed):
Some issues:
* The indentation must be four spaces, you have changed it to tabulators
* You've moved the member variables into the `_Ready` method in C#, that's invalid C# AFAIK, they shouldn't have `private`, or leading underscore
* And I agree with the above, it should just be a simpler example

DarkiStar committed:
[Doc] Update Crypto example

removed node and _ready()
removed C# private and underscore
reformated to 4 spaces
added some newlines for better separation

DarkiStar committed:
Fighting with spaces

Hopefully this time the indentions are correct.

AThousandShips commented on doc/classes/Crypto.xml:
```diff
@@ -1,75 +1,76 @@
+        # Save key and certificate in the user folder.
+        key.save("user://generated.key")
+        cert.save("user://generated.crt")
+            
```
```suggestion

```

AThousandShips commented:
It should be tabs for everything except code indentation, as it was originally

DarkiStar commented:
Github Editor is kind of difficult ...
I will make another try. Tabs for everything except code. But it looks strange ...

AThousandShips commented:
The first two indentations for the code should be tabs, like it was originally 

I'd suggest using an editor on a computer, you're going to have to squash your commits anyway before this can be merged and it's much easier than the web editor

DarkiStar committed:
Tabs / Spaces fix

Not sure if code allways needs 4 spaces ...

DarkiStar committed:
Tab / spaces Fix 2

Another try

AThousandShips commented on doc/classes/Crypto.xml:
```diff
@@ -5,69 +5,71 @@
 		The Crypto class provides access to advanced cryptographic functionalities.
-		Currently, this includes asymmetric key encryption/decryption, signing/verification, and generating cryptographically secure random bytes, RSA keys, HMAC digests, and self-signed [X509Certificate]s.
+		Currently, this includes asymmetric key encryption/decryption, signing/verification, and generating
+		cryptographically secure random bytes, RSA keys, HMAC digests, and self-signed [X509Certificate]s.
```
This line shouldn't be broken up into two

DarkiStar committed:
Linebreak fix

Another try :-)

AThousandShips reviewed (approved):
Makes sense and looks good to me

Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

DarkiStar commented:
Allready commited my crypto branch to my godot master fork. So there are no differences between my crypto branch and master. Not sure how to continue.

AThousandShips commented:
You should reset your master branch to the upstream branch and then rebase it, or rebase against the upstream branch

DarkiStar commented:
Thank you for your patience. Can I somehow trigger the rebase/reset of the master branch directly on GitHub, or does there necessarily have to be a copy on my computer to initiate these functions?

AThousandShips commented:
You need to do this on a computer, you can't squash in the web interface

DarkiStar committed:
Merge branch 'master' into DarkiStar-Crypto

DarkiStar commented:
I would like to manage it on my own.
